src/main/java/com/taskmanagement/tasks/TaskService.java

```diff
@@ -0,0 +1,160 @@
+package com.taskmanagement.tasks;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TaskService implements ITaskService {
+    @Autowired
+    private final ITaskRepo taskRepo;
+
+    public TaskService(ITaskRepo taskRepo) {
+        this.taskRepo = taskRepo;
+    }
+
+    @Override
+    public List<TaskDto> findAll() {
+        List<TaskDto> taskDtos = new ArrayList<>();
+        for (Task task : taskRepo.findAll()) {
+            taskDtos.add(TaskConverter.Converter(task));
+        }
+        return taskDtos;
+    }
+    @Override
+    public List<TaskDto> findByPoint(Integer point) {
```

Comment:
I don't think you should search a task by creating many functions like this (findByPoint, findByStatus...).
How about for multiple search?